scripts/watch.js

```diff
@@ -123,7 +123,7 @@ const setupPreloadPackageWatcher = ({ws}) =>
    const folderName = path.resolve(__dirname, '../extensions/' + name);
 
   console.log('dirname is', folderName);
-    spawnProcess = spawn('yarn', ['--cwd',  folderName, 'watch'] );
+    spawnProcess = spawn('yarn', ['--cwd',  folderName, 'watch'] , { shell: process.platform == 'win32'});
```

Comment:
not sure what `==` will do and why we have something for windows there ?

Reply:
it set `shell = true` if script running on windows, we need that to be able run that script on windows

Reply:
ok but we should need three `=`, `===`

Reply:
and if need it only for windows, we should have options

like
```
options = {};
if (process.platform == 'win32') {
options.shell = true;
}
```

so we don't have shell variables on others operating system

Reply:
`isWin` exists as `util`